src/lang/english.txt

```diff
@@ -4789,6 +4790,8 @@ STR_ERROR_FOREST_CAN_ONLY_BE_PLANTED                            :{WHITE}... fore
 STR_ERROR_CAN_ONLY_BE_BUILT_ABOVE_SNOW_LINE                     :{WHITE}... can only be built above the snow-line
 STR_ERROR_CAN_ONLY_BE_BUILT_BELOW_SNOW_LINE                     :{WHITE}... can only be built below the snow-line
 
+STR_ERROR_PROSPECTING_WAS_UNLUCKY                               :{WHITE}Prospecting was unlucky and failed; try again
```

Comment:
Not sure about the wording here - it's still unclear that the action cost money - if you're not looking at your bank balance, you might not notice...

Reply:
Maybe "You were unlucky; your funds failed to prospect for a new industry" or something like that? I'm really sure how to word this so I'll take any suggestions?

Reply:
I think I have now resolved this in [this diff](https://github.com/OpenTTD/OpenTTD/compare/8433ce22b64220d69d19bd84d9c30f4b54b23cb5..17f79c5018e97feb128b9dd979f4a8568260a7e9), it should now be clear that the action has cost money and that it failed because of random chance. If I've missed anything else in the wording let me know.